main.go

```diff
@@ -108,7 +108,7 @@ func readOauthToken() string {
 	return strings.TrimSuffix(s, "\n")
 }
 
-func listPRs(client *github.Client) []*github.PullRequest {
+func listPRs(client *github.Client, startTime time.Time) []*github.PullRequest {
 	prs := make([]*github.PullRequest, 0)
```

Comment:
```suggestion
	prs := make([]*github.PullRequest, 10_000)
```

Essentially, let's just allocate way more than enough memory up front.

Reply:
My mistake, should have been:

```suggestion
	prs := make([]*github.PullRequest, 0, 10_000)
```